packages/bun-types/sqlite.d.ts

```diff
@@ -1040,4 +1112,21 @@ declare module "bun:sqlite" {
      */
     readonly byteOffset: number;
   }
+
+  /**
+   * An object representing the changes made to the database since the last `run` or `exec` call.
+   *
+   * @since Bun v1.1.14
+   */
+  interface Changes {
+    /**
+     * The number of rows changed by the last `run` or `exec` call.
+     */
+    changes: number;
+
+    /**
+     * If `safeIntegers` is `true`, this is a `bigint`. Otherwise, it is a `number`.
+     */
+    lastInsertRowId: number | bigint;
```

Comment:
`better-sqlite3` uses `id` instead of `Id`. Do we want to at least support an alias?

Reply:
yeah